lalrpop/src/lr1/codegen/ascent.rs

```diff
@@ -317,6 +317,7 @@ impl<'ascent, 'grammar, W: Write> CodeGenerator<'ascent,
             for (index, &token) in tokens.iter().enumerate() {
                 let pattern = match token {
                     Token::Terminal(s) => format!("Some({})", self.match_terminal_pattern(s)),
+                    Token::Error => unimplemented!(),
```

Comment:
BTW, I'm fine with landing this without support in the recursive ascent style -- but I'd prefer to give a structured error. =)